checkov/common/checks_infra/solvers/attribute_solvers/base_attribute_solver.py

```diff
@@ -41,6 +44,21 @@ def run(self, graph_connector: DiGraph) -> Tuple[List[Dict[str, Any]], List[Dict
         return passed_vertices, failed_vertices
 
     def get_operation(self, vertex: Dict[str, Any]) -> bool:  # type:ignore[override]
+        if self.is_jsonpath_check and self.attribute:
```

Comment:
Can the entire logic under the if be extracted to a method?

Reply:
done